components/datetime/src/helpers.rs

```diff
@@ -14,41 +14,38 @@
 ///
 /// The size should correspond to the Rust version in rust-toolchain.toml.
 ///
-/// If the size on latest nightly differs from rust-toolchain.toml, use the
+/// If the size on latest beta differs from rust-toolchain.toml, use the
 /// named arguments version of this macro to specify both sizes:
 ///
 /// ```ignore
-/// size_test!(MyType, my_type_size, pinned = 32, nightly = 24);
+/// size_test!(MyType, my_type_size, pinned = 32, beta = 24);
 /// ```
 ///
 /// The test is ignored by default but runs in CI. To run the test locally,
 /// run `cargo test -- --include-ignored`
 macro_rules! size_test {
-    ($ty:ty, $id:ident, pinned = $pinned:literal, nightly = $nightly:literal) => {
+    ($ty:ty, $id:ident, pinned = $pinned:literal, beta = $beta:literal) => {
         macro_rules! $id {
             () => {
                 concat!(
                     "📏 This item has a stack size of <b>",
                     stringify!($pinned),
-                    " bytes</b> in the default ICU4X Rust stable toolchain and <b>",
-                    stringify!($nightly),
-                    " bytes</b> on nightly at release date."
+                    " bytes</b> on the stable toolchain and <b>",
+                    stringify!($beta),
+                    " bytes</b> on beta toolchain at release date."
```

Comment:
Nightly, beta, and stable could all be different sizes. I believe it was recently that they actually were all different at the same time. Right now this test only looks at stable and nightly. If we want it to also look at beta, we should change the macro to allow a beta size and a nightly size.

Reply:
Yes but that would give us 4 different combinations which I don't all want to copy paste right now. Can we cross that bridge when we get there?

Reply:
Just make two cases, all the same and all explicit.

Reply:
(this is what we currently have: just make the all-different case have three instead of two different values)